package/yast2-registration.spec

```diff
@@ -47,6 +47,7 @@ Requires:       yast2-add-on >= 3.1.8
 # packager/product_patterns.rb
 Requires:       yast2-packager >= 3.1.95
 Requires:       yast2-update >= 3.1.36
+Requires:	autoyast2-installation
```

Comment:
We should avoid this dependency, this will pull whole AY stack into the installed system where is does not makes sense. Autoyast is listed as a [skelcd-control dependency](https://github.com/yast/skelcd-control-leanos/blob/master/package/skelcd-control-leanos.spec#L48) so it will be included in inst-sys anyway.